test/integration/discovery.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+load helpers
+
+DISCOVERY_FILE=/tmp/test_file_discovery
```

Comment:
`mkfile` is not available everywhere. Maybe I'll use `cat /dev/null > /tmp/test_file_discovery` to create an empty file.